jobs/ops.rb

```diff
@@ -13,7 +13,7 @@
 TM_HOSTNAME = ENV['TM_HOSTNAME'] || 'tm'
 TM_PORT = ENV['TM_PORT'] || '80'
 
-TM_SWAGGER_SPEC = 'https://' + ENV['TM_HOSTNAME'] + ':' + TM_PORT + '/swagger/v1/swagger.json'
+TM_SWAGGER_SPEC = 'https://' + TM_HOSTNAME + ':' + TM_PORT + '/swagger/v1/swagger.json'
```

Comment:
Use string interpolation: `TM_SWAGGER_SPEC = "https://#{TM_HOSTNAME}:#{TM_PORT}/swagger/v1/swagger.json"`

Reply:
yep, missed that, thanks John